src/index.d.ts

```diff
@@ -29,6 +29,8 @@ interface UserVars {
 
 type LogLevel = 'log' | 'info' | 'warn' | 'error' | 'debug';
 
+type VarScope = 'document' | 'evt' | 'page' | 'user';
```

Comment:
Do we have required fields for all of these scopes? I only see page vars in the help docs, maybe we need to document these too, and would it be worthy to strongly type them, for example `page` would require `pageName` + any additional properties?

Reply:
[I was referring to these properties](https://github.com/cowpaths/mn/blob/9e50b6b642835ee386a72cb8dfc6c45ec6fb8e6b/projects/fullstory/packages/recording-api/src/index.ts#L1120), although I was only able to find a help article about setting page vars. It also looks like [we also only consume `user` for the varScope](https://github.com/cowpaths/mn/blob/ffde541ed0f746451445e24f12221d49d7666bcd/projects/fullstory/go/src/fs/bundlestore/metadata.go#L82). There's a comment that says: `// We only care about "user" scope vars for now`. 

Should I go ahead and remove 'evt', and 'document' and add another example for 'user' scope in README.md? 

I didn't want to strongly type them because someone can upload page properties without a page name.

Reply:
As per this thread here: the setVar for the SDK will only support 'the 'page' VarScope' .

https://fullstory.slack.com/archives/C2CK80L1M/p1643142326323900?thread_ts=1643140203.321900&cid=C2CK80L1M